elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexWriteTransaction.java

```diff
@@ -194,46 +190,4 @@ public Object loadObject(Class<?> entityClass,
         DataStoreTransaction transaction = getTransaction(entityClass);
         return hold(transaction, transaction.loadObjects(entityClass, filterExpression, sorting, pagination, scope));
     }
-
-    @Override
-    public <T> Object getRelation(
```

Comment:
Hmm.  We removed all the defaults.  I get that since these are deprecated, but I feel like there should be new versions of these functions that implement the getRelation functions that take FilterExpressions - for correctness.